packages/auth/src/authClients/nhost.ts

```diff
@@ -0,0 +1,43 @@
+import type { NhostClient, User } from 'nhost-js-sdk'
+
+export type Nhost = NhostClient
+export type NhostUser = User
+
+import { AuthClient } from './'
+
+export const nhost = (client: Nhost): AuthClient => {
```

Comment:
Can you please provide an interface that extends AuthClient and use that instead? That way we can get more specific types. 
Have a look at supabase auth for an example

Reply:
@Tobbe I will do this later tonight or tomorrow.

Reply:
@Tobbe just pushed those changes. Let me know if there is anything missing.